lib/styles/darkenColors.js

```diff
@@ -0,0 +1,19 @@
+import Color from 'color';
```

Comment:
@shivam-daksh `'color'` package will need to be added to the `package.json`. Also then please run `yarn install` and commit both `package.json` and `yarn.lock. Without it, it won't work for others :) Thanks!

Reply:
Hey @MisRob, I first tried `npm install color` and `yarn add color` , both doesn't worked. 
So, I added `"color": "4.2.3"` in **package.json** file in `dependencies` manually and again fired `yarn install`. But it is giving me errors that my node version should be greater than `">=12.5.0"` however I'm using 10.24.1 as per documentation (10.x required).

<img width="1124" alt="image" src="https://github.com/user-attachments/assets/c89ffdbb-98f7-4f44-82ad-b93ffa99fbb7">

Reply:
Your development environment is set up correctly. This means that this newest `color` package version requires Node >= 12.5.0. 

We can't upgrade our Node quite yet so we need to stay with Node v10. Could you try to install the highest version of the `color` package that still works with Node 10? It seems they restricted it in https://github.com/Qix-/color/releases/tag/4.2.1 so https://github.com/Qix-/color/releases/tag/4.2.0 may be a good candidate to try out. After you installed, please check that your implementation still works as expected.

Reply:
I'm not sure if this will work though because looking more at the changelogs, it seems that in `4.2.1` they only started communicating about the requirement for Node v12 that was there for longer time before.. https://github.com/Qix-/color/pull/236.

Reply:
Well, let's see if we can figure something out!

If not, we could wait for merging this PR until our Node v18 upgrade https://github.com/learningequality/kolibri-design-system/pull/645 is ready :)

Reply:
Till then, I'm trying the other versions of `color` that are compatible with node@10.

Reply:
Hey @MisRob , `color` version 3.2.1 is working fine as expected. Shoult I go with it?

Reply:
Great, thanks for researching that. Yes, let's give it a try. Could you pin it to this exact version in the package.json? So that we don't upgrade by mistake until we have Node 18.